### ISSUE TITLE
BLANK_LINE_BETWEEN_PROPERTIES: Misleading error message

### ISSUE THREAD

**Issue by petertrr** [diktat-static-analysis/diKTat#1044]
## Describe the bug
```
.../DownloadFilesTest.kt:62:5: [BLANK_LINE_BETWEEN_PROPERTIES] there should be no blank lines between properties without comments; comment or KDoc on property should have blank line before: fileSystemRepository
```
while in reality the code is as follows:
```kotlin
@Autowried
fileSystemRepository
```
so instead of KDoc there is an annotation

## Environment information
* diktat version: 1.0.0-rc.3

**Comment by sanyavertolet** [comment]
Do you have any snippet of code to reproduce the bug?

**Comment by petertrr** [comment]
Yes, here:
```kotlin
class A {
    val bar: Bar
    @Autowried
    val fileSystemRepository: Foo
}
```
there is a misleading erorr message regarding missing blank line before `@Autowired`

**Comment by sanyavertolet** [comment]
@petertrr do you want me just to edit warning message in `Warnings.kt` or you would like to see something like that:
```
[BLANK_LINE_BETWEEN_PROPERTIES] there should be no blank lines between properties without comments; comment, KDoc or annotation on property should have blank line before: @Autowried
```

**Comment by petertrr** [comment]
> 

I think, warning is enough. It would point to the correct line, and if mention annotation in the message, it will become more clear. We don't output parts of KDocs here, so shouldn't output annotations either.
